scripts/prox_crosswalk.py

```diff
@@ -42,18 +42,17 @@ def getSourceIDs(keyID, sourcesList, identifiers):
     foundSources = _getCachedIDsForPlace(keyID, sourcesList)
     toFetch = [source for source in sourcesList if source not in foundSources]
     foundSources.update(fetchAndCacheSources(keyID, toFetch, identifiers))
-    # TODO: fetch from each source, and error if there is a problem
-    # TODO: return list of UPDATED sources?
+    return foundSources
 
 def fetchAndCacheSources(keyID, sourcesList, identifiers):
     newSources = {}
     coordinates = (identifiers["lat"], identifiers["lng"])
     name = identifiers["name"]
     for source in sourcesList:
         if source == "tripadvisor":
-            res = ta.search(coordinates, name)
+            res = ta.search(coordinates, name)["data"]
```

Comment:
Not you, but I wonder if ta.search should just index directly into data – it doesn't look like there are any other valuable keys